frontend/public/components/overview/pods-overview.tsx

```diff
@@ -154,32 +159,38 @@ export const PodsOverviewContent: React.SFC<PodsOverviewContentProps> = ({
   const errorPodCount = _.size(_.filter(pods, (pod) => isPodError(pod)));
   const podsShown = Math.max(Math.min(errorPodCount, MAX_ERROR_PODS), MAX_PODS);
   const linkTo = allPodsLink || `${resourcePath(referenceFor(obj), name, namespace)}/pods`;
-  const emptyMessage = emptyText || 'No Pods found for this resource.';
+  const emptyMessage = emptyText || t('public~No Pods found for this resource.');
+
+  const PodAlert = () =>
+    showWaitingForBuildAlert ? (
+      <Alert
+        isInline
+        variant="info"
+        title={t('public~Waiting for the build')}
+        actionLinks={
+          <AlertActionLink onClick={() => setShowWaitingPods(!showWaitingPods)}>
+            {showWaitingPods
+              ? t('public~Hide waiting pods with errors')
+              : t('public~Show waiting pods with errors')}
+          </AlertActionLink>
+        }
+      >
+        {t(
+          'public~Waiting for the first build to run successfully. You may temporarily see "ImagePullBackOff" and "ErrImagePull" errors while waiting.',
+        )}
+      </Alert>
+    ) : null;
```

Comment:
No need to create a function here, you can just write jsx directly.
```suggestion
  const PodAlert = showWaitingForBuildAlert ? (
      <Alert
        isInline
        variant="info"
        title={t('public~Waiting for the build')}
        actionLinks={
          <AlertActionLink onClick={() => setShowWaitingPods(!showWaitingPods)}>
            {showWaitingPods
              ? t('public~Hide waiting pods with errors')
              : t('public~Show waiting pods with errors')}
          </AlertActionLink>
        }
      >
        {t(
          'public~Waiting for the first build to run successfully. You may temporarily see "ImagePullBackOff" and "ErrImagePull" errors while waiting.',
        )}
      </Alert>
    ) : null;
```